libvast/CMakeLists.txt

```diff
@@ -318,7 +318,8 @@ target_link_libraries(libvast PUBLIC ${ARROW_LIBRARY})
 dependency_summary("Apache Arrow" ${ARROW_LIBRARY} "Dependencies")
 
 # Link against Boost.
-target_link_libraries(libvast PRIVATE Boost::headers)
+target_link_libraries(libvast PRIVATE Boost::headers Boost::filesystem)
+target_link_libraries(libvast_builtins PRIVATE Boost::headers Boost::filesystem)
```

Comment:
I believe you need to add `libboost-filesystem1.81-dev` to `scripts/debian/install-dev-dependencies.sh`. Not sure what needs to be done on the macOS side.

Reply:
I'll push fixes for both.

Reply:
I think it needs a tad more CMake massaging:

```
-- Generating done
  The link interface of target "vast::libvast_builtins" contains:

    Boost::headers

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.

Call Stack (most recent call first):
  /__w/vast/vast/_install/lib/cmake/vast/VASTConfig.cmake:84 (include)
  CMakeLists.txt:15 (find_package)


CMake Generate step failed.  Build files cannot be regenerated correctly.
Error: Process completed with exit code 1.
```

Reply:
I pushed a commit that should fix this.

Reply:
Works like a charm!